docs/sources/developers/plugins/publish-a-plugin.md

```diff
@@ -180,6 +76,53 @@ To submit an **update** for an already published plugin:
      - Select **Multiple** if you'd like to submit separate plugin archives for each architecture.
        This can lead to faster downloads since users can select the specific architecture they want to install the plugin on.
    - **URL:** A URL that points to a ZIP archive of your packaged plugin.
-   - **Source Code URL:** A URL that points to a public git repository or ZIP archive of your complete plugin source code.
+   - **Source Code URL:** A URL that points to a public Git repository or ZIP archive of your complete plugin source code. See [examples](#what-source-code-url-formats-are-supported).
    - **MD5:** The MD5 hash of the plugin specified by the **URL**.
 1. Click **Submit**.
+
+## Frequently asked questions
+
+### Do I need to submit a private plugin?
```

Comment:
Feel like the titles and ordering of the FAQ could use some extra love, but we can also do it in any other phase